notebooks/TFSegFormer_Finetune.ipynb

```diff
@@ -1,15 +1,5 @@
 {
```

Comment:
<p>The -&gt; This</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-diver/segformer-tf-transformers/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>